seesaw.ts

```diff
@@ -101,8 +265,7 @@ Implementation Notes
     }
 
     export class Seesaw {
-        _drdy: DigitalPin;
```

Comment:
will this _drdy pin feature not work with microbit?
it's not connected by default on the the crickit boards, but may become useful or necessary in the future.
If it was removed for a reason that's fine, but if it's just because it was unused could we leave it in there?

Reply:
It was removed because pin representation are incompatible between Adafruit and Microbit. I have replaced it with a function callback so that this detail can be abstracted upstream (until we fix this pin stuff).